src/gui/owncloudsetupwizard.cpp

```diff
@@ -346,7 +346,7 @@ void OwncloudSetupWizard::slotAuthError()
     }
 
     _ocWizard->show();
-    if (_ocWizard->currentId() == WizardCommon::Page_ShibbolethCreds) {
+    if (_ocWizard->currentId() == WizardCommon::Page_ShibbolethCreds || _ocWizard->currentId() == WizardCommon::Page_HttpCreds) {
```

Comment:
Why? Isn't it okay to stay on the HttpCreds page if auth fails?

Also, isn't this unrelated to OAuth?